tslint.json

```diff
@@ -148,6 +149,7 @@
     "use-pipe-transform-interface": true,
     "pipe-naming": [true, "camelCase", "sky"],
     "component-class-suffix": true,
-    "directive-class-suffix": true
+    "directive-class-suffix": true,
+    "no-focused-tests": true
```

Comment:
it looks like `tslint-jasmine-rules` can do more than the `no-focused-tests`. do we want the others?

```json
{
  "no-focused-tests": true,
  "no-disabled-tests": true,
  "expect-length": true
}
```

Reply:
I thought about the `no-disabled-tests` but I know in Java land we sometimes have to disable a test (often pact) for some time. `tslint` allows you configure a rule to be on but only a warning. We could do that for `no-disabled-tests`. I haven't looked into what `expect-length` does.

Reply:
My vote is to isolate this particular pull request to `no-focused-tests`, since I know of a few folks using disabled tests to keep a running "backlog" of tests they need to write, but haven't yet.
```
xit('should write this test when I have time', () => {});
```

Reply:
I added `no-disabled-tests` as a warning. I'm assuming that warnings don't prevent things from building, releasing, etc. I tried `skyux test`, `skyux lint`, `skyux server` and `skyux watch` with the warning turned on and everything worked. If does cause problems I'm happy to remove it.